### PR TITLE
Enhancement/issue 1 init project setup

### LINKED ISSUE #1: 1) Init Project Setup
As defined in the milestone objectives:

a) unit testing
b) linting
c) project dotfiles (gitignore, gitattributes, .editorconfig, .yo-rc.json
d) package.json
e) CONTRIBUTING.md
f) README.md
g) Vagrant
h) build scripts (build.sh)

cc: @paul-barry-kenzan

### PR DESCRIPTION
**Pull request by obuckley-kenzan**
resolves #1 

cc: @paul-barry-kenzan

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
ok to test

File: bin/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+echo "installing npm dependencies"
+rm -rf node_modules > /dev/null 2>&1
+npm cache clean
+
+npm install
```
Comment (paul-barry-kenzan) [inline bin/build.sh]: Should this have the condition to install Bower, if `bower.json` exists? 

https://github.com/ThomsonReuters-IPS/generator-webui/blob/master/generators/common/templates/_build.sh#L12

Comment (obuckley-kenzan) [inline bin/build.sh]: I don't see bower fitting into the build time of generator-keystone itself.  I was thinking when it comes time to actually generating a seed, there will be a template specfic `vagrant.sh` to use there.  Does that work with you?

Comment (paul-barry-kenzan) [inline bin/build.sh]: For this repo itself, you're probably right. Bower isn't necessary. Good to go.

**Pull request merged**